crates/ironrdp-pdu/src/rdp/headers.rs

```diff
@@ -262,6 +262,7 @@ pub enum ShareDataPdu {
     FrameAcknowledge(FrameAcknowledgePdu),
     ServerSetErrorInfo(ServerSetErrorInfoPdu),
     Input(InputEventPdu),
+    ShutdownRequest,
```

Comment:
Nice! You don’t have to address the client-side immediately, but I had an issue related to that: https://github.com/Devolutions/IronRDP/issues/115